packages/cli/package.json

```diff
@@ -7,6 +7,8 @@
     "node": ">=14.15.1"
   },
   "dependencies": {
+    "@amplitude/identify": "^1.9.0",
+    "@amplitude/node": "^1.9.0",
```

Comment:
Can we remove `coveo.analytics`?

Reply:
@olamothe, do you see any issue with replacing Coveo UA? or you think we should wait until we have Amplitude data in prod

Reply:
I think it's okay if we remove coveo.analytics, just make sure that it does not crash/there's no error before doing so.

That's the most important part.